storage/rocksdb/rdb_datadic.cc

```diff
@@ -4853,6 +4965,53 @@ Rdb_index_stats Rdb_dict_manager::get_stats(GL_INDEX_ID gl_index_id) const {
   return Rdb_index_stats();
 }
 
+rocksdb::Status
+Rdb_dict_manager::put_auto_incr_val(rocksdb::WriteBatchBase *batch,
+                                    const GL_INDEX_ID &gl_index_id,
```

Comment:
How is GL_INDEX_ID defined? Small enough for pass by value?

Reply:
Yeah, it is a struct with 2 32 bit unsigneds, so plenty small enough that pass by reference is probably less efficient than just a stack push.